pkg/dev/amazonlinux2016.09/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN mkdir -p /etc/osquery /var/log/osquery /etc/hubble/hubble.d /opt/hubble /opt
 #osquery should be built first since requirements for other packages can interfere with osquery dependencies
 #to build, osquery scripts want sudo and a user to sudo with.
 #to pin to a different version change the following envirnment variable
-ENV OSQUERY_SRC_VERSION=1ed050147ac88a7314c172e15e86f8886206dd06
+ENV OSQUERY_SRC_VERSION=0314871908273b2770129da9f5a7c3206f224f68
```

Comment:
Dockerfile changes are not yet finalized, any changes in dockerfiles just reflect what could potentially change.